examples/pytorch/model_zoo/geometric/mnist.py

```diff
@@ -16,7 +16,7 @@
 from coordinate import get_coordinates, z2polar
 
 argparser = argparse.ArgumentParser("MNIST")
-argparser.add_argument("--gpu", type=int, default=-1,
+argparser.add_argument("--gpu", type=int, default=5,
```

Comment:
-1 in default indicates running on cpu is preferable. 5 is probably a unavailable gpu index.

Reply:
Yes. Sure